components/QuestChain/QuestChainV1Page.tsx

```diff
@@ -961,8 +1013,8 @@ export const QuestChainV1Page: React.FC<QuestChainV1PageProps> = ({
                       </Flex>
                       {numSubmissionsToReview > 0 && (
                         <NextLink
-                          as={`/chain/${questChain.chainId}/${questChain.address}/review`}
-                          href={`/chain/[chainId]/[address]/review`}
+                          as={`/${questChain.chainId}/${questChain.address}/review`}
```

Comment:
```suggestion
                          as={`/${AVAILABLE_NETWORK_INFO[questChain.chainId].urlName}/${questChain.address}/review`}
```

Reply:
There are a few other places where we need to fix this i think

Reply:
think i covered all the cases now. i just left the v0 quest chains